### ISSUE TITLE
Error when referencing restsharp newer than 105.2.3.0

### ISSUE THREAD

**Issue by xklonx** [mundipagg/OneSignal-CSharp-SDK#29]
I've updated restsharp nuget to 106.3.1 and i'm recieving error
System.IO.FileLoadException: Could not load "RestSharp, Version=105.2.3.0, Culture=neutral, PublicKeyToken=null"
after OneSignalClient constructor call.
<dependentAssembly>
		    <assemblyIdentity name="RestSharp" publicKeyToken="null" culture="neutral"/>
		    <bindingRedirect oldVersion="0.0.0.0-106.3.1.0" newVersion="106.3.1.0"/>
	    </dependentAssembly>
This redirect not helps.

**Comment by xklonx** [comment]
No, I just copied the project to my solution, also to use this future https://github.com/mundipagg/OneSignal-CSharp-SDK/pull/23. Last commit more than year ago, no one seems to be updating this project, which is a pity.

**Comment by JimmyPun610** [comment]
Check this out
Update to RestSharp 106 and added filter by external Id
Required .net framework 4.6 or more
https://github.com/JimmyPun610/OneSignal-CSharp-RestApiSDK
